requirements.txt

```diff
@@ -1,6 +1,7 @@
 # Requirements for building PEPs with Sphinx
-sphinx >= 3.5
-docutils >= 0.16
+Pygments ~= 2.9.0
+Sphinx ~= 4.0.2
+docutils ~= 0.17.1
 
```

Comment:
Only question I'd have is if better to pin to compatible releases or all future? (`~=` vs `>=`) - historically been unpinned so I think that means the latest release has always been used (but I am not best placed to make this call, as no idea if e.g. previously there have been breaking changes to PEP rendering due to dependency changes, or the risk appetite for this in the future vs new features that may or may not be used)

A

Reply:
Okay, changed it back